docs/configuration-reference/components/metallb.md

```diff
@@ -84,14 +83,15 @@ Table of all the arguments accepted by the component.
 
 Example:
 
-| Argument                    | Description                                                                                | Default | Required |
-|-----------------------------|--------------------------------------------------------------------------------------------|:-------:|:--------:|
-| `address_pools`             | A map which allows specifying one or more CIDRs which MetalLB can use to expose services.  | -       | true     |
-| `controller_node_selectors` | A map with specific labels to run MetalLB controller pods selectively on a group of nodes. | -       | false    |
-| `speaker_node_selectors`    | A map with specific labels to run MetalLB speaker pods selectively on a group of nodes.    | -       | false    |
-| `controller_toleration`     | Specify one or more tolerations for controller pods.                                       | -       | false    |
-| `speaker_toleration`        | Specify one or more tolerations for speaker pods.                                          | -       | false    |
-| `service_monitor`           | Create ServiceMonitor for Prometheus to scrape MetalLB metrics.                            | false   | false    |
+| Argument                    | Description                                                                                | Default | Type                                                                                                           | Required |
+|-----------------------------|--------------------------------------------------------------------------------------------|:-------:|:---------------------------------------------------------------------------------------------------------------|:--------:|
+| `address_pools`             | A map which allows specifying one or more CIDRs which MetalLB can use to expose services.  |    -    | object({default = list(string), special_addresses = list(string)})                                             |   true   |
+| `controller_node_selectors` | A map with specific labels to run MetalLB controller pods selectively on a group of nodes. |    -    | map(string)                                                                                                    |  false   |
```

Comment:
I find weird that this is defined as `map(string)` instead of `map(string->string)` or similar but I guess that's how terraform rolls...

Nothing to do, just a comment.